Run Information
========================
cognizant.intelligent.test.scripter.engine : 2.0
java.runtime.name    : Java(TM) SE Runtime Environment
java.version         : 1.8.0_281
java.home            : C:\Program Files\Java\jdk1.8.0_281\jre
os.name              : Windows 10
os.arch              : amd64
os.version           : 10.0
file.encoding        : UTF-8
========================
Run Started on Tue Apr 26 14:33:52 IST 2022
Loading Browser Profile
Loading RunManager
[Scenario:Web] [TestCase: Login1] [Description: Login1] [Browser: Chrome Headless] [BrowserVersion: Default] [Platform: ANY][ExistingBrowser: false]
----------------------------------------------------------
[1] TestCase selected for execution from [//MyRelease/WebSet]
----------------------------------------------------------
Initializing Report
ApiLink not available!!
testset : WebSet
Run Manager true
Continue Execution true
Shutting Down Thread as there is no need for SelectiveThreading
Running Iteration 1
Launching Chrome Headless
Launching Local Driver
Starting ChromeDriver 98.0.4758.102 (273bf7ac8c909cde36982d27f66f3c70846a3718-refs/branch-heads/4758@{#1151}) on port 35115
Only local connections are allowed.
Please see https://chromedriver.chromium.org/security-considerations for suggestions on keeping ChromeDriver safe.
ChromeDriver was started successfully.
2022-04-26 14:33:54:04S +0530 [INFO] org.openqa.selenium.remote.ProtocolHandshake createSession:Detected dialect: W3C
Chrome Headless Launched
Step:1   | Object: Browser | Action: Open | Input: @https://www.saucedemo.com/ | Conditon:  | @26-Apr-2022 14:33:54.054
[DONE]   | Opened Url: https://www.saucedemo.com/

===================================================================================================

Step:2   | Object: UserID | Action: Set | Input: @standard_user | Conditon:  | @26-Apr-2022 14:33:55.055
Using @id [user-name], Object 'UserID' Found in 17 ms
[DONE]   | Entered Text 'standard_user' on 'UserID'

===================================================================================================

Step:3   | Object: Password | Action: Set | Input: @test | Conditon:  | @26-Apr-2022 14:33:55.055
Using @id [password], Object 'Password' Found in 18 ms
[DONE]   | Entered Text 'test' on 'Password'

===================================================================================================

Step:4   | Object: [Login] Button | Action: Click | Input:  | Conditon:  | @26-Apr-2022 14:33:55.055
Using @id [login-button], Object '[Login] Button' Found in 12 ms
[DONE]   | Clicking on [Login] Button

===================================================================================================

Step:5   | Object: Browser | Action: takeScreenshot | Input:  | Conditon:  | @26-Apr-2022 14:33:55.055
[PASS]   | Screenshot is taken

===================================================================================================


---------------------------------------------------
Testcase Name        : Web:Login1
Executed Steps       : 5
Passed Steps         : 5
Failed Steps         : 0
Time Taken           : 00:00:04
-----------------------------------------------------

2022-04-26 14:33:57:04S +0530 [SEVERE] com.cognizant.cognizantits.engine.reporting.sync.AbstractHttpClient parseResponse:null
Unexpected token END OF FILE at position 0.
	at org.json.simple.parser.JSONParser.parse(JSONParser.java:257)
	at org.json.simple.parser.JSONParser.parse(JSONParser.java:81)
	at org.json.simple.parser.JSONParser.parse(JSONParser.java:75)
	at com.cognizant.cognizantits.engine.reporting.sync.AbstractHttpClient.parseResponse(AbstractHttpClient.java:160)
	at com.cognizant.cognizantits.engine.reporting.sync.BasicHttpClient.Get(BasicHttpClient.java:391)
	at com.cognizant.cognizantits.engine.reporting.sync.BasicHttpClient.Get(BasicHttpClient.java:341)
	at com.cognizant.cognizantits.engine.reporting.sync.azure.AzureClient.isConnected(AzureClient.java:121)
	at com.cognizant.cognizantits.engine.reporting.sync.azure.AzureSync.isConnected(AzureSync.java:60)
	at com.cognizant.cognizantits.engine.reporting.SummaryReport.updateTMResults(SummaryReport.java:116)
	at com.cognizant.cognizantits.engine.reporting.SummaryReport.updateTestCaseResults(SummaryReport.java:110)
	at com.cognizant.cognizantits.engine.core.Task.run(Task.java:94)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

2022-04-26 14:33:57:04S +0530 [SEVERE] com.cognizant.cognizantits.engine.reporting.sync.azure.AzureClient isConnected:null
java.lang.NullPointerException
	at com.cognizant.cognizantits.engine.reporting.sync.azure.AzureClient.isConnected(AzureClient.java:122)
	at com.cognizant.cognizantits.engine.reporting.sync.azure.AzureSync.isConnected(AzureSync.java:60)
	at com.cognizant.cognizantits.engine.reporting.SummaryReport.updateTMResults(SummaryReport.java:116)
	at com.cognizant.cognizantits.engine.reporting.SummaryReport.updateTestCaseResults(SummaryReport.java:110)
	at com.cognizant.cognizantits.engine.core.Task.run(Task.java:94)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

[ERROR:UNABLE TO REACH TEST MANAGEMENT MODULE!!!]
[DEBUG]   | Unable to Connect to TM Module

===================================================================================================

Run Finished on Tue Apr 26 14:33:57 IST 2022
-----------------------------------------------------
ExecutionDate        : 26-Apr-2022 14-33-52
Executed TestCases   : 1
Passed TestCases     : 1
Failed TestCases     : 0
Time Taken           : 00:00:05
ReportPath           : file:///C:\Users\586126\Work\CITS\CITS2.0\Gitlab\Java8\cognizant-intelligent-test-scripter-java8\Dist\release\Projects\TestProject\Results\TestExecution\MyRelease\WebSet\26-Apr-2022 14-33-52\summary.html
-----------------------------------------------------
Latest Excel Report Path C:\Users\586126\Work\CITS\CITS2.0\Gitlab\Java8\cognizant-intelligent-test-scripter-java8\Dist\release\Projects\TestProject\Results\TestExecution\MyRelease\WebSet\Latest
-----------------------------------------------------
EXCEL REPORT CREATED SUCCESSFULLY
-----------------------------------------------------
2022-04-26 14:33:59:04S +0530 [INFO] com.cognizant.cognizantits.engine.reporting.impl.excel.ExcelSummaryHandler launchexcel:Trying To Open Excel
2022-04-26 14:33:59:04S +0530 [INFO] com.cognizant.cognizantits.engine.reporting.impl.excel.ExcelSummaryHandler launchexcel:Opened Excel Report Successfully
2022-04-26 14:33:59:04S +0530 [INFO] com.cognizant.cognizantits.engine.reporting.sync.azure.AzureClient createNewTestRun:Conneting Azure DevOps to update results
2022-04-26 14:33:59:04S +0530 [SEVERE] com.cognizant.cognizantits.engine.reporting.sync.azure.AzureClient createNewTestRun:null
java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.rangeCheck(ArrayList.java:659)
	at java.util.ArrayList.get(ArrayList.java:435)
	at com.cognizant.cognizantits.engine.reporting.sync.azure.AzureClient.createNewTestRun(AzureClient.java:174)
	at com.cognizant.cognizantits.engine.reporting.sync.azure.AzureSync.disConnect(AzureSync.java:69)
	at com.cognizant.cognizantits.engine.core.Control.endExecution(Control.java:198)
	at com.cognizant.cognizantits.engine.core.Control.startRun(Control.java:179)
	at com.cognizant.cognizantits.engine.core.Control.start(Control.java:62)
	at com.cognizant.cognizantits.engine.core.Control.call(Control.java:71)
	at com.cognizant.cognizantits.ide.main.mainui.EngineConfig.runProject(EngineConfig.java:46)
	at com.cognizant.cognizantits.ide.main.mainui.components.testexecution.testset.TestsetComponent$18.run(TestsetComponent.java:278)
	at java.lang.Thread.run(Thread.java:748)

